src/elements/option/option/option-base-element.ts

```diff
@@ -193,6 +197,12 @@ abstract class SbbOptionBaseElement extends SbbDisabledMixin(
         this.updateAriaDisabled();
       } else if (mutation.attributeName === 'data-negative') {
         this.negative = this.hasAttribute('data-negative');
+      } else {
```

Comment:
I wonder if the name of this method is still correct with the updated use case.
This would be a breaking change, but should we rename this to e.g. `onExternalMutation`?